pkg/xray/watches.go

```diff
@@ -184,9 +185,38 @@ func packAssignedPolicies(policies []WatchAssignedPolicy) []interface{} {
 	return l
 }
 
+func packWatch(watch Watch, d *schema.ResourceData) diag.Diagnostics {
+	if err := d.Set("description", watch.GeneralData.Description); err != nil {
+		return diag.FromErr(err)
+	}
+	if err := d.Set("active", watch.GeneralData.Active); err != nil {
+		return diag.FromErr(err)
+	}
+	if err := d.Set("watch_resource", packProjectResources(watch.ProjectResources)); err != nil {
+		return diag.FromErr(err)
+	}
+	if err := d.Set("assigned_policy", packAssignedPolicies(watch.AssignedPolicies)); err != nil {
+		return diag.FromErr(err)
+	}
+	return nil
+}
+
+var retryOnMergeError = func() func(response *resty.Response, _r error) bool {
```

Comment:
This should be in the `util.go` since it's used by both files

Reply:
yes, you're right, fixed!